sparkle/src/lottie/spinnerColorXS.ts

```diff
@@ -1,3 +1,325 @@
+const animation = {
+  v: "5.9.0",
```

Comment:
Are these diffs expected? These file are automatically generated I believe?

Reply:
Updates on lottie files were due to prettier changing files it shouldn't have. I've dropped that commit 👍